.github/elemental-iso-add-registration

```diff
@@ -36,7 +36,7 @@ iso_repack() {
 
   case ${ISO_URL} in
     http*)
-      wget -O "${TEMPDIR}/iso/${ISO_FILE}" "${ISO_URL}"
+      curl -o "${TEMPDIR}/iso/${ISO_FILE}" "${ISO_URL}"
```

Comment:
`download.opensuse.org/repositories/...` will be redirected to `ftp.gwdg.de/pub/opensuse/repositories/`-
So, you mean the curl -L option here :-) 
```suggestion
      curl -Lo "${TEMPDIR}/iso/${ISO_FILE}" "${ISO_URL}"
```